packages/react-ui-core/src/Gmap/OverlayView.js

```diff
@@ -116,13 +121,19 @@ export default class OverlayView extends PureComponent {
       // Projection is undefined if map has not yet initialized
       if (!projection) return
 
-      const { x, y } = projection.fromLatLngToDivPixel(position)
+      const { x, y } = this.overlay.getProjection().fromLatLngToDivPixel(position)
```

Comment:
why not just use `projection` here? no need to call `this.overlay.getProjection()` again... is there?